requirements/common.txt

```diff
@@ -398,8 +398,8 @@ mysqlclient==2.0.1 \
     --hash=sha256:edd42ccaa444b00702d5374b2f5f7585c9d0ce201917f15339f1c3cf91c1b1ed \
     --hash=sha256:fb2f75aea14722390d2d8ddf384ad99da708c707a96656210a7be8af20a2c5e5 \
     # via -r requirements/common.in
-newrelic==5.14.1.144 \
-    --hash=sha256:340ebcdb0dd08bfb597c71598d6d8f746a93f7e4921f10b3616c9142c608a14d \
+newrelic==5.2.1.129 \
+    --hash=sha256:da9adab674d9fe7aa8fbabb6691b33fb7be94a32b6a80548ec7018be9df8ef65 \
```

Comment:
This deploys to prototype:  I will continue to see if I can get prototype to fail deployment with another branch